sycl/test/regression/fsycl-save-temps.cpp

```diff
@@ -22,4 +22,4 @@ int main() {
 }
 
 // TODO: Address a Windows-specific issue with integration header filenames
-// XFAIL: system-windows
+// UNSUPPORTED: system-windows
```

Comment:
`save-temps` is supposed to be working on windows as well.
Please, differentiate Windows shells by checking one of the triples available in the config object.
https://github.com/intel/llvm/blob/sycl/llvm/test/lit.cfg.py#L100

Reply:
Reverted change